wp-web-push/readme.txt

```diff
@@ -6,19 +6,30 @@ Tested up to: 4.4.1
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
-Web Push notifications for your website.
+Send push notifications to your visitors! Uses the W3C Push API.
+
+== Description ==
+This plugin uses the [W3C Push API](https://www.w3.org/TR/push-api/) to allow you to send push notifications to anyone who visits your site and signs up for notifications.
+
+Once you\'ve installed this plugin, anyone visiting your site/app in [browsers that support the W3C Push API](http://caniuse.com/#feat=push-api) will be asked whether he/she wants to receive notifications from your site. You will immediately be able to send push notifications to anyone who consents.
```

Comment:
> Let's reference platatus instead of caniuse :smile:

I wanted to but platatus doesn't have information for mobile browsers which I think is probably super important.

> Can you also add something about the configuration of the subscription behavior and the analytics? I'm not sure how much detailed.

Sure! I couldn't figure out how to configure the plugin though. Can you ping me with that info?